x-pack/plugins/aiops/public/components/change_point_detection/split_field_selector.tsx

```diff
@@ -9,63 +9,71 @@ import React, { FC, useMemo, useCallback } from 'react';
+    const { splitFieldsOptions } = useChangePointDetectionControlsContext();
 
-  const options = useMemo<EuiComboBoxOptionOption[]>(() => {
-    return [
-      {
-        value: undefined,
-        label: i18n.translate('xpack.aiops.changePointDetection.notSelectedSplitFieldLabel', {
-          defaultMessage: '--- Not selected ---',
-        }),
-      },
-      ...splitFieldsOptions.map((v) => ({
-        value: v.name,
-        label: v.displayName,
-        ...(v.name ? { field: { id: v.name, type: v?.type } } : {}),
-      })),
-    ];
-  }, [splitFieldsOptions]);
+    const options = useMemo<EuiComboBoxOptionOption[]>(() => {
+      return [
+        {
+          value: undefined,
+          label: i18n.translate('xpack.aiops.changePointDetection.notSelectedSplitFieldLabel', {
+            defaultMessage: '--- Not selected ---',
+          }),
+        },
+        ...splitFieldsOptions.map((v) => ({
+          value: v.name,
+          label: v.displayName,
+          ...(v.name ? { field: { id: v.name, type: v?.type } } : {}),
+        })),
+      ];
+    }, [splitFieldsOptions]);
+
+    const selection = options.filter((v) => v.value === value);
 
-  const selection = options.filter((v) => v.value === value);
+    const onChangeCallback = useCallback(
+      (selectedOptions: EuiComboBoxOptionOption[]) => {
+        const option = selectedOptions[0];
+        const newValue = option?.value as string;
+        onChange(newValue);
+        if (closeFlyout) {
+          closeFlyout();
+        }
+      },
+      [onChange, closeFlyout]
+    );
 
-  const onChangeCallback = useCallback(
-    (selectedOptions: EuiComboBoxOptionOption[]) => {
-      const option = selectedOptions[0];
-      const newValue = option?.value as string;
-      onChange(newValue);
-      closeFlyout();
-    },
-    [onChange, closeFlyout]
-  );
+    const label = i18n.translate('xpack.aiops.changePointDetection.selectSpitFieldLabel', {
+      defaultMessage: 'Split field',
+    });
 
-  return (
-    <EuiFormRow>
-      <EuiComboBox
-        compressed
-        prepend={i18n.translate('xpack.aiops.changePointDetection.selectSpitFieldLabel', {
-          defaultMessage: 'Split field',
-        })}
-        singleSelection={{ asPlainText: true }}
-        options={options}
-        selectedOptions={selection}
-        onChange={onChangeCallback}
-        isClearable
-        data-test-subj="aiopsChangePointSplitField"
-        // @ts-ignore
-        renderOption={renderOption}
-      />
-    </EuiFormRow>
-  );
-});
+    return (
+      <EuiFormRow fullWidth label={inline ? undefined : label}>
+        <EuiComboBox
+          fullWidth
+          compressed
+          prepend={inline ? label : undefined}
+          singleSelection={{ asPlainText: true }}
+          options={options}
+          selectedOptions={selection}
+          onChange={onChangeCallback}
+          isClearable
+          data-test-subj="aiopsChangePointSplitField"
+          // @ts-ignore
```

Comment:
Nit: would be good to know what the ignore is for

Reply:
removed in https://github.com/elastic/kibana/pull/163694/commits/fdbbbba41b5ee621e42692c72bfb49bb3fdb45b9